tests/responses/test_fileresponse.py

```diff
@@ -31,6 +38,45 @@ def test_file_from_buffer_unnamed_attachment(self):
         self.assertEqual(response['Content-Disposition'], 'attachment')
         self.assertEqual(list(response), [b'binary content'])
 
+    def test_file_from_buffer_absolute_name(self):
+        response = FileResponse(io.BytesIO(b'binary content'), filename=__file__)
+        self.assertEqual(response['Content-Length'], '14')
+        self.assertIn(response['Content-Type'], ['text/x-python', 'text/plain'])
+        self.assertEqual(response['Content-Disposition'], 'inline; filename="test_fileresponse.py"')
+        self.assertEqual(list(response), [b'binary content'])
+
+    def test_file_from_named_buffer1(self):
```

Comment:
Combine with `subTest()` 

Is it worth adding the _relative_ named versions?

Reply:
IDK if it's worth right now, as it shouldn't affect anything anymore, but it doesn't hurt either, so i've added it for the future.